nixops/backends/__init__.py

```diff
@@ -322,11 +343,11 @@ def get_keys(self):
     def get_ssh_name(self):
         assert False
 
-    def get_ssh_flags(self, scp=False):
+    def get_ssh_flags(self, scp=False) -> List[str]:
```

Comment:
Nice.

Reply:
I looked at making this _get_ssh_flags, but other plugins (aws at least) defines the function. Maybe it shouldn't, though.